apps/spruce/src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -602,5 +607,7 @@ const GitHubChecksAliasesDescription = (
     </StyledLink>{" "}
     in this project&rsquo;s config YAML instead if Version Control is enabled
     and no aliases are defined on the project or repo page.
+    {projectType === ProjectType.Repo &&
+      "If enabled, these settings can only apply to one branch project that also has this feature enabled (and does not apply to untracked branches)."}
```

Comment:
[nit] Same as above if you want
```suggestion
      "If enabled, these settings can only apply to one branch project that also has this feature enabled. They do not apply to untracked branches."}
```